src/pages/groupComparison/GroupComparisonMutationMapper.tsx

```diff
@@ -46,14 +51,101 @@ export default class GroupComparisonMutationMapper extends MutationMapper<
+                    }}
+                >
+                    <div style={{ fontWeight: 'bold', paddingBottom: 3 }}>
+                        (A){' '}
+                        {
+                            this.store.dataStore.sortedFilteredGroupedData[0]
+                                .group
+                        }
+                    </div>
+                    <DriverAnnotationProteinImpactTypeBadgeSelector
+                        filter={this.proteinImpactTypeFilter}
+                        counts={this.mutationCountsByProteinImpactTypeForGroup(
+                            0
+                        )}
+                        onSelect={this.onProteinImpactTypeSelect}
+                        onClickSettingMenu={this.props.onClickSettingMenu}
+                        annotatedProteinImpactTypeFilter={
+                            this.annotatedProteinImpactTypeFilter
+                        }
+                    />
+                    <hr style={{ marginTop: 10, marginBottom: 10 }}></hr>
+                    <div style={{ fontWeight: 'bold', paddingBottom: 3 }}>
+                        (B){' '}
+                        {
+                            this.store.dataStore.sortedFilteredGroupedData[1]
+                                .group
+                        }
+                    </div>
+                    <DriverAnnotationProteinImpactTypeBadgeSelector
+                        filter={this.proteinImpactTypeFilter}
+                        counts={this.mutationCountsByProteinImpactTypeForGroup(
+                            1
+                        )}
+                        onSelect={this.onProteinImpactTypeSelect}
+                        onClickSettingMenu={this.props.onClickSettingMenu}
+                        annotatedProteinImpactTypeFilter={
+                            this.annotatedProteinImpactTypeFilter
+                        }
+                    />
+                </div>
+            </div>
+        );
+    }
+
+    protected mutationsGroupedByProteinImpactTypeForGroup(groupIndex: number) {
+        // there are two types of filters (with putative driver, without putative driver)
+        const filtersWithoutProteinImpactTypeFilter = this.store.dataStore.dataFilters.filter(
+            f =>
+                f.type !== DataFilterType.PROTEIN_IMPACT_TYPE &&
+                f.type !== ANNOTATED_PROTEIN_IMPACT_FILTER_TYPE
+        );
+
+        // apply filters excluding the protein impact type filters
+        // this prevents number of unchecked protein impact types from being counted as zero
+        let sortedFilteredGroupedData = applyDataFilters(
+            this.store.dataStore.sortedFilteredGroupedData[groupIndex].data,
+            filtersWithoutProteinImpactTypeFilter,
+            this.store.dataStore.applyFilter
         );
+
+        // also apply lazy mobx table search filter
+        sortedFilteredGroupedData = sortedFilteredGroupedData.filter(m =>
```

Comment:
is this necessary in this feature, since there is no filtering?

Reply:
not necessary unless we decide to enable the table later